pkg/common/common.go

```diff
@@ -60,12 +54,9 @@ const (
 	AzurePluginImage     = "quay.io/konveyor/velero-plugin-for-microsoft-azure:latest"
 	GCPPluginImage       = "quay.io/konveyor/velero-plugin-for-gcp:latest"
 	CSIPluginImage       = "quay.io/konveyor/velero-plugin-for-csi:latest"
-	VSMPluginImage       = "quay.io/konveyor/velero-plugin-for-vsm:latest"
-	// DataMoverImage is the data mover controller for data mover CRs - VolumeSnapshotBackup and VolumeSnapshotRestore
-	DataMoverImage      = "quay.io/konveyor/volume-snapshot-mover:latest"
-	DummyPodImage       = "quay.io/konveyor/rsync-transfer:latest"
-	RegistryImage       = "quay.io/konveyor/registry:latest"
-	KubeVirtPluginImage = "quay.io/konveyor/kubevirt-velero-plugin:v0.2.0"
+	DummyPodImage        = "quay.io/konveyor/rsync-transfer:latest"
```

Comment:
DummyPodImage is also for 1.2 datamover, so you should remove all references to it as well.